### DISCUSSION TITLE
Merge Conflict

### DISCUSSION THREAD

**Discussion by Malinga14** [community/community#145170] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to merge my feature branch into the main branch, but I'm getting a merge conflict. What should I do?

**Comment by SadeepaNHerath** [comment, marked as answer]
To resolve a merge conflict, follow these steps:

1.Identify the Conflict: Git will indicate which files have conflicts when you try to merge. Open those files in your code editor.
2.Resolve the Conflict: Look for the conflict markers (<<<<<<<, =======, >>>>>>>) in the files. Decide how to combine the changes from both branches, and remove the conflict markers.
3.Stage the Changes: Once you've resolved the conflicts, save the files and run git add <filename> to stage the changes.
4.Complete the Merge: Finally, run git commit to complete the merge.
